aten/src/ATen/native/sparse/cuda/SparseCsrTensorMath.cu

```diff
@@ -706,6 +706,84 @@ struct ReductionMulOp {
   __forceinline__ scalar_t identity_cpu() const { return 1; }
 };
 
+void _apply_sparse_csr_linear_solve(
+  const Tensor& A,
+  const Tensor& b,
+  const bool left, 
+  const Tensor& x) {
+#if defined(USE_ROCM) || !defined(USE_CUDSS)
+  TORCH_CHECK(
+      false,
+      "Calling torch.linalg.solve with sparse tensors requires compiling ",
+      "PyTorch with CUDA and not supported in ROCm build.");
+#else
+  TORCH_CHECK(A.is_sparse_csr(), "A must be a CSR matrix");
+  TORCH_CHECK(b.dim() == 1, "b must be a 1D tensor");
```

Comment:
Have you hit a problem with a matrix for `b`? cuDSS documentation says the right-hand-side argument can be a matrix.

Reply:
I haven't encountered before, but I'll stay tuned on any future update on cuDSS.